docs/src/user_docs/assembly/io_operations.md

```diff
@@ -12,11 +12,11 @@ Miden assembly provides a set of instructions for moving data between the operan
 | ------------------------------------------------------------------------- | ----------- | ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | push.*a* <br> - *(1-2 cycles)* <br> push.*a*.*b* <br> push.*a*.*b*.*c*... | [ ... ]     | [a, ... ] <br> [b, a, ... ] <br> [c, b, a, ... ] | Pushes values $a$, $b$, $c$ etc. onto the stack. Up to $16$ values can be specified. All values must be valid field elements in decimal (e.g., $123$) or hexadecimal (e.g., $0x7b$) representation. |
 
-When specifying values in hexadecimal format, it is possible to omit the periods between individual values as long as total number of specified bytes is a multiple of $8$. That is, the following are semantically equivalent:
+The value can be specified in hexadecimal format without periods between individual values as long as it specifying a Word, i.e. total number of specified bytes in the value is equal to $32$. Note that hexadecimal values devided by periods (short hexadecimal strings) assumed to be encoded in big-endian format, while the strings specifying the whole Word without periods (long hexadecimal strings) assumed to be encoded in little-endian format. That is, the following are semantically equivalent:
 
 ```
-push.0x1234.0xabcd
-push.0x0000000000001234000000000000abcd
+push.0x00001234.0x00005678.0x00009012.0x0000abcd
+push.0x123400000000000056780000000000009012000000000000abcd000000000000
```

Comment:
I would also add a version here where the values are in decimal form (separated by periods).